src/api/db/migrate/20220518094002_increase_event_id_column_capacity.rb

```diff
@@ -0,0 +1,9 @@
+class IncreaseEventIdColumnCapacity < ActiveRecord::Migration[6.1]
+  def up
+    safety_assured { change_column :events, :id, :bigint, auto_increment: true }
+  end
+
+  def down
+    safety_assured { change_column :events, :id, :integer, auto_increment: true }
```

Comment:
This is irreversible IMHO.

Reply:
I wonder if it's better to raise an exception or simply do nothing.

```
  def down
    raise ActiveRecord::IrreversibleMigration
  end
```
VS 

```
  def down; end
```

Reply:
I would raise the exception

Reply:
https://guides.rubyonrails.org/active_record_migrations.html#using-reversible